storage/src/print_bucket_default_acl.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * Copyright 2022 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * For instructions on how to run the full sample:
+ *
+ * @see https://github.com/GoogleCloudPlatform/php-docs-samples/tree/master/storage/README.md
+ */
+
+namespace Google\Cloud\Samples\Storage;
+
+# [START storage_print_bucket_default_acl]
+use Google\Cloud\Storage\StorageClient;
+
+/**
+ * Print all entities and roles for a bucket's Default ACL.
+ *
+ * @param string $bucketName The name of your Cloud Storage bucket.
+ */
+function print_bucket_default_acl($bucketName)
```

Comment:
```suggestion
function print_bucket_default_acl(string $bucketName): void {
```

Use [this](https://github.com/GoogleCloudPlatform/php-docs-samples/blob/master/storage/src/create_bucket_notifications.php#L37-L40) as a reference. Same for the other 2 PRs.